api-audit/src/main/java/com/capitalone/dashboard/service/AuditServiceImpl.java

```diff
@@ -339,21 +372,7 @@ private void getRelatedCommits(String aCommit, HashMap<String, Commit> mapCommit
             allPeerReviews.add(peerReviewResponse);
 
             //check to see if pr was reviewed
-            List<Comment> comments = pr.getComments();
-            boolean peerReviewed = false;
-            for(Comment comment: comments) {
-                if (!comment.getUser().equalsIgnoreCase(prAuthor)) {
-                    peerReviewed = true;
-                    break;
-                }
-            }
-            List<Comment> reviewComments = pr.getReviewComments();
-            for(Comment comment: reviewComments) {
-                if (!comment.getUser().equalsIgnoreCase(prAuthor)) {
-                    peerReviewed = true;
-                    break;
-                }
-            }
+            boolean peerReviewed = computePeerReviewStatus(pr);
```

Comment:
@mspiegel The check for the comments and reviewComments needs to be added as a fallback if the above compute returns false.